docs/installation_configuration_guide/DeploymentManagerConfiguration.md

```diff
@@ -292,9 +292,11 @@ Deployment Manager of type `lite-embedded` has the following configuration optio
-| shouldVerifyBeforeDeploy  | boolean  | true          | By default, before redeployment of scenario with state from savepoint, verification of savepoint compatibility is performed. There are some cases when it can be too time consuming or not possible. Use this flag to disable it. |
-| shouldCheckAvailableSlots | boolean  | true          | When set to true, Nussknacker checks if there are free slots to run new job. This check should be disabled on Flink Kubernetes Native deployments, where Taskmanager is started on demand.                                        |
+| Parameter                           | Type     | Default value | Description                                                                                                                                                                                                                       |
+|-------------------------------------|----------|---------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| restUrl                             | string   |               | The only required parameter, REST API endpoint of the Flink cluster                                                                                                                                                               |
+| jobManagerTimeout                   | duration | 1 minute      | Timeout for communication with FLink cluster. Consider extending if e.g. you have long savepoint times etc.                                                                                                                       |
+| shouldVerifyBeforeDeploy            | boolean  | true          | By default, before redeployment of scenario with state from savepoint, verification of savepoint compatibility is performed. There are some cases when it can be too time consuming or not possible. Use this flag to disable it. |
+| shouldCheckAvailableSlots           | boolean  | true          | When set to true, Nussknacker checks if there are free slots to run new job. This check should be disabled on Flink Kubernetes Native deployments, where Taskmanager is started on demand.                                        |
+| waitForDuringDeployFinish.maxChecks | boolean  | 60            | After Flink job execution, we check if tasks were successfully started on TaskMangers. This parameter describe how many times we should check it before notifying about deployment failure.                                       |
+| waitForDuringDeployFinish.delay     | boolean  | 1 second      | After Flink job execution, we check if tasks were successfully started on TaskMangers. This parameter describe how long should be delay between checks.                                                                           |
```

Comment:
Isn't it too small? Maybe 5/10s between each check?

Reply:
I'm not sure about that number - I would like to provide to user the quickest response. Maybe we can check how it will work and increase it if it will be not ok. WDYT?